OpenCorePkg.dsc

```diff
@@ -375,8 +386,10 @@
   #
   !include NetworkPkg/NetworkComponents.dsc.inc
 
+  CryptoPkg/Library/IntrinsicLib/IntrinsicLib.inf
+
 [LibraryClasses]
-  NULL|OpenCorePkg/Library/OcCompilerIntrinsicsLib/OcCompilerIntrinsicsLib.inf
+  NULL|CryptoPkg/Library/IntrinsicLib/IntrinsicLib.inf
```

Comment:
cc @savvamitrofanov @vit9696 - I believe it is very important to support HTTPS boot as well as HTTP boot, therefore we need to also ship TlsDxe. However this depends on CryptoPkg/Library/IntrinsicLib/IntrinsicLib.inf. We cannot just specify this library in the `!if $(NETWORK_TLS_ENABLE) == TRUE` code block above, because it conflicts without our library (defines some of the same externs), and our intrinsics library is set to be added to everything.

I have currently changed to use the CryptoPkg intrinsics library for everything, instead. All seems to work fine, AFAICT, but might perhaps be running slower than necessary in order (e.g.) to be making constant time guarantees, or something. Or might be other negatives I am not aware of.

We could try to update our intrinsics library to add the couple of externs which are missing for it to link with CryptoPkg, but suspect this would be a bad idea (for same reason, i.e. CryptoPkg library might be addressing e.g. constant time security concerns which we are not otherwise bothered with).

I could otherwise update to adding our library manually, wherever it is actually needed, rather than globally; so that CrypoPkg library can be used just in CryptoPkh, without conflicting with ours.

Reply:
As discussed, will try modifying so that everything uses more or less our intrinsics lib, however updated with missing required functions, and moved as core library to MdePkg in audk.